superset/datasets/commands/duplicate.py

```diff
@@ -44,49 +43,39 @@
 
 class DuplicateDatasetCommand(CreateMixin, BaseCommand):
     def __init__(self, data: Dict[str, Any]) -> None:
-        self._base_model: SqlaTable = SqlaTable()
+        self._base_model: Optional[SqlaTable] = None
         self._properties = data.copy()
+        self._database: Optional[Database] = None
 
     def run(self) -> Model:
         self.validate()
         try:
-            database_id = self._base_model.database_id
             table_name = self._properties["table_name"]
             owners = self._properties["owners"]
-            database = db.session.query(Database).get(database_id)
-            if not database:
-                raise SupersetErrorException(
-                    SupersetError(
-                        message=__("The database was not found."),
-                        error_type=SupersetErrorType.DATABASE_NOT_FOUND_ERROR,
-                        level=ErrorLevel.ERROR,
-                    ),
-                    status=404,
-                )
             table = SqlaTable(table_name=table_name, owners=owners)
-            table.database = database
-            table.schema = self._base_model.schema
-            table.template_params = self._base_model.template_params
+            table.database = self._database  # type: ignore
```

Comment:
Addressing review comments, we are now leaning for readability.

Reply:
@Antonio-RiveroMartnez thanks for this! I've asked @betodealmeida to see if he can suggest any other patterns here, since now based on my feedback there are a lot of `ignore`s. We talked about validating sooner, or moving some of the validation logic into the init. If there are any suggestions that you have to make this cleaner, we can propose a new pattern here.